### PR TITLE
Spoi 9322 database to hdfs app template

### PR DESCRIPTION
**Pull request by yogidevendra**

### CODE REVIEW

**Comment by yogidevendra** [comment]
I will squash the commits once all review comments are incorporated.

**Comment by yogidevendra** [comment]
@DT-Priyanka @amberarrow  Could you please review this?

**Comment by amberarrow** [comment]
Ok, it'll be a couple of days though.

File: tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java [hunk 1]
```diff
@@ -47,31 +47,42 @@ public void populateDAG(DAG dag, Configuration conf)
...
+    
+    /*
+     * To add custom logic to your DAG, add your custom operator here with
+     * dag.addOperator api call and connect it in the dag using the dag.addStream
+     * api call. 
+     * 
+     * For example: 
+     * 
+     * To add the transformation operator in the DAG, use the following block of
+     * code below.
```
Comment (amberarrow) [inline tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java]: below => omit

File: tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java [hunk 2]
```diff
@@ -47,31 +47,42 @@ public void populateDAG(DAG dag, Configuration conf)
...
+     * 
+     * TransformOperator transform = dag.addOperator("Transform", new TransformOperator());
+     * Map<String, String> expMap = Maps.newHashMap();
+     * expMap.put("name", "{$.name}.toUpperCase()");
+     * transform.setExpressionMap(expMap);
+     * 
+     * And to connect it in the DAG as follows,
```
Comment (amberarrow) [inline tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java]: follows, => follows:

File: tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java [hunk 3]
```diff
@@ -47,31 +47,42 @@ public void populateDAG(DAG dag, Configuration conf)
...
+     * JdbcPoller --> Transform --> Formatter --> FileOutput
+     *
+     * Replace the following line below
```
Comment (amberarrow) [inline tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java]: line below => line:

File: tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java [hunk 4]
```diff
@@ -47,31 +47,42 @@ public void populateDAG(DAG dag, Configuration conf)
...
+     * dag.addStream("dbrecords", poller.outputPort, formatter.in);
+     * 
+     * with the following two lines
```
Comment (amberarrow) [inline tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java]: lines => lines:

File: tutorials/README.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 | Application       | Description |
 | ----------------- | ----------- |
+| database-to-hdfs  | Ingest records from a DataBase table to hadoop HDFS.  |
```
Comment (amberarrow) [inline tutorials/README.md]: Out of order (maintain lexicographic order)

File: tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+public class Application implements StreamingApplication
+{
+  public void populateDAG(DAG dag, Configuration conf)
+  {
+    JdbcPOJOPollInputOperator poller = dag.addOperator("JdbcPoller", new JdbcPOJOPollInputOperator());
+
+    JdbcStore store = new JdbcStore();
+    poller.setStore(store);
+    poller.setFieldInfos(addFieldInfos());
+
```
Comment (amberarrow) [inline tutorials/database-to-hdfs/src/main/java/com/datatorrent/apps/Application.java]: Trailing whitespace {several files have this issue}

**Comment by yogidevendra** [comment]
This app package will be moved to app-templates repo. Hence closing this PR.

**Pull request closed without merging**